models/quality_measures/intermediate/nqf0059_diabetes_hemoglobin_a1c/quality_measures__int_nqf0059_denominator.sql

```diff
@@ -0,0 +1,178 @@
+    , coalesce(claim_end_date,claim_start_date) as max_date
+    from {{ref('quality_measures__stg_medical_claim')}} medical_claim
+    inner join {{ref('quality_measures__int_nqf0059__performance_period')}}  as pp on
+        coalesce(claim_end_date,claim_start_date)  >=  pp.performance_period_begin
+         and coalesce(claim_start_date,claim_end_date) <=  pp.performance_period_end
+    inner join  visit_codes
+        on medical_claim.hcpcs_code= visit_codes.code
+
+
+)
+
+, all_encounters as (
+    select *, 'v' as visit_enc,cast(null as {{ dbt.type_string() }}) as proc_enc, cast(null as {{ dbt.type_string() }}) as claim_enc
+    from visits_encounters
+    union all
+    select *, cast(null as {{ dbt.type_string() }}) as visit_enc, 'p' as proc_enc, cast(null as {{ dbt.type_string() }}) as claim_enc
+    from procedure_encounters
+    union all
+    select *, cast(null as {{ dbt.type_string() }}) as visit_enc,cast(null as {{ dbt.type_string() }}) as proc_enc, 'c' as claim_enc
+    from claims_encounters
+)
+
+, encounters_by_patient as (
+    select patient_id,min(min_date) min_date, max(max_date) max_date,
+        concat(concat(
+            coalesce(min(visit_enc),'')
+            ,coalesce(min(proc_enc),''))
+            ,coalesce(min(claim_enc),'')
+            ) as qualifying_types
+    from all_encounters
+    group by patient_id
+)
+
+, diabetics_codes as (
+
+    select
+          code
+        , code_system
+    from {{ ref('quality_measures__value_sets') }}
+    where lower(concept_name) in (
+          'diabetes'
+        , 'hba1c laboratory test'
+    )
+)
+
+, conditions as (
+
+    select
+          patient_id
+        , claim_id
+        , encounter_id
+        , recorded_date
+        , source_code
+        , source_code_type
+        , normalized_code
+        , normalized_code_type
+    from {{ ref('quality_measures__stg_core__condition')}}
+
+)
+
+, diabetic_conditions as (
+
+    select
+          conditions.patient_id
+        , conditions.claim_id
+        , conditions.encounter_id
+        , conditions.recorded_date
+        , conditions.source_code
+        , conditions.source_code_type
+    from conditions
+    inner join diabetics_codes
+        on conditions.source_code_type = diabetics_codes.code_system
+            and conditions.source_code = diabetics_codes.code
+
+)
+
+, patients_with_age as (
+    select
+          p.patient_id
+        , min_date
+        , floor({{ datediff('birth_date', 'e.min_date', 'hour') }} / 8760.0)  as min_age
+        , max_date
+        , floor({{ datediff('birth_date', 'e.max_date', 'hour') }} / 8760.0) as max_age
+        , qualifying_types
+    from {{ref('quality_measures__stg_core__patient')}} p
+    inner join encounters_by_patient e
+        on p.patient_id = e.patient_id
+    where p.death_date is null
+
+
+)
+
+, qualifying_patients as (
+
+    select
+        distinct
+          diabetic_conditions.patient_id
+        , patients_with_age.* exclude patient_id
```

Comment:
Sorry, I missed this in my first review. Can you change this to list out the columns from patients_with_age?

Reply:
exclude clause has been removed